CONTRIBUTING.md

```diff
@@ -1,62 +1,73 @@
+Once you have built your template file(s), the commit message should have one of the following forms:
 
-* Use the following for newly added packages: ```New package: <pkgname>-<version>```
-  [Example](https://github.com/void-linux/void-packages/commit/176d9655429188aac10cd229827f99b72982ab10)
+* for new packages, use ```New package: <pkgname>-<version>``` ([example](https://github.com/void-linux/void-packages/commit/176d9655429188aac10cd229827f99b72982ab10)).
 
-* Use the following if you update a package: ```<pkgname>: update to <version>.```
-  [Example](https://github.com/void-linux/void-packages/commit/b6b82dcbd4aeea5fc37a32e4b6a8dd8bd980d5a3)
+* for package updates, use ```<pkgname>: update to <version>.``` ([example](https://github.com/void-linux/void-packages/commit/b6b82dcbd4aeea5fc37a32e4b6a8dd8bd980d5a3)).
 
-* If you changed something on the template without a version change use ```<pkgname>: <reason>```
-  [Example](https://github.com/void-linux/void-packages/commit/8b68d6bf1eb997cd5e7c095acd040e2c5379c91d)
+* for template modifications without a version change, use ```<pkgname>: <reason>``` ([example](https://github.com/void-linux/void-packages/commit/8b68d6bf1eb997cd5e7c095acd040e2c5379c91d)).
 
-If you want to describe your changes in more detail, make an empty line and add the description afterwards.
-[Example](https://github.com/void-linux/void-packages/commit/f1c45a502086ba1952f23ace9084a870ce437bc6)
+If you want to describe your changes in more detail, add an empty line followed by those details ([example](https://github.com/void-linux/void-packages/commit/f1c45a502086ba1952f23ace9084a870ce437bc6)).
 
-This is also described in the [manual](https://github.com/void-linux/void-packages/blob/master/Manual.md) in deeper detail.
+For further information, please consult the [manual](https://github.com/void-linux/void-packages/blob/master/Manual.md).
 
-There's a helper tool for committing packages in the [xtools](https://github.com/chneukirchen/xtools) package, which can be used when adding or updating a package:
+`xbump`, available in the [xtools](https://github.com/chneukirchen/xtools) package, can be used to commit a new or updated package:
 
     $ xbump <pkgname> <git commit options>
 
 `xbump` will use `git commit` to commit the changes with the appropriate commit message. For more fine-grained control over the commit, specific options can be passed to `git commit` by adding them after the package name. 
 
+After committing your changes, please check that the package builds successfully. From the top level directory of your local copy of the `void-packages` repository, run:
+
+    $ ./xbps-src pkg <pkgname>
+    
+Your package must build successfully for at least x86, but we recommend trying to build for armv* as well, e.g.:
+
+    $ ./xbps-src -a armv7l pkg <pkgname>
+    
+For further details, see the output of `./xbps-src --help`.
```

Comment:
unrecognized option

Reply:
Ah, i was fooled by `xbps-src` (very reasonably) showing help output when presented with an unrecognised option. :-) i'll change it to `-h`.